docs/hooks/useScaffoldEventHistory.md

```diff
@@ -16,14 +16,17 @@ const {
   eventName: "GreetingChange",
   // Specify the starting block number from which to read events, this is a bigint.
   fromBlock: 31231n,
-  blockData: true,
+  // If set to true, the events will be updated every pollingInterval milliseconds set at scaffoldConfig (default: false)
+  watch: true,
   // Apply filters to the event based on parameter names and values { [parameterName]: value },
   filters: { premium: true },
+  // If set to true it will return the block data for each event (default: false)
+  blockData: true,
   // If set to true it will return the transaction data for each event (default: false),
   transactionData: true,
   // If set to true it will return the receipt data for each event (default: false),
   receiptData: true,
 });
 ```
 
-This example retrieves the historical event logs for the `GreetingChange` event of the `YourContract` smart contract, starting from block number 31231 and filtering events where the premium parameter is true. The data property of the returned object contains an array of event objects, each containing the event parameters and (optionally) the block, transaction, and receipt data. The `isLoading` property indicates whether the event logs are currently being fetched, and the `error` property contains any error that occurred during the fetching process (if applicable).
+This example retrieves the historical event logs for the `GreetingChange` event of the `YourContract` smart contract, starting from block number 31231 and filtering events where the premium parameter is true. The data property of the returned object contains an array of event objects, each containing the event parameters and (optionally) the block, transaction, and receipt data. The `isLoading` property indicates whether the event logs are currently being fetched, and the `error` property contains any error that occurred during the fetching process (if applicable). Since `watch` is set to true, the event logs will be refetched every `pollingInterval` milliseconds (set at scaffoldConfig.config.ts).
```

Comment:
In `useScaffoldEventSubscriber.md` we are adding a link to the details of the [`pollingInterval`](/deploying/deploy-nextjs-app#--pollinginterval) parameter in case we see it interesting here also.  
In case of adding the link, maybe "_(set at scaffoldConfig.config.ts)_" could be deleted. If not, we could change the style for `scaffoldConfig.config.ts` to match other places of the docs where we mention filenames.

Reply:
Thanks Pablo, agree!

Shiv already fixed it https://github.com/scaffold-eth/se2-docs/pull/38/commits/1314b2a1141cb66e9f89c7d5b2a7a969c7192d61 :cowboy_hat_face:

Reply:
🙌 I actually kept both because experienced people just need to know where they need to update `pollingInterval` without following the `pollingInterval` link. 

But beginners would like to follow the link and get description about `pollingInterval` if its their first time 🙌